test/e2e/conftest.py

```diff
@@ -44,8 +90,259 @@ def pytest_collection_modifyitems(config, items):
 # Provide a k8s client to interact with the integration test cluster
 @pytest.fixture(scope='class')
 def k8s_client():
-    return k8s._get_k8s_api_client()
+    return k8sclient._get_k8s_api_client()
 
 @pytest.fixture(scope='module')
 def lambda_client():
-    return boto3.client('lambda', region_name=get_region())
\ No newline at end of file
+    return boto3.client('lambda', region_name=get_region())
+
+@pytest.fixture
```

Comment:
Pytest fixtures should be defined in whichever scope they're used. I defined `lambda_client` and `k8s_client` in this file because they can be used by every/any test. These fixtures below are specific to individual tests (function, code signing, etc.). I'd prefer if we moved these to the `test_*.py` files. If we need to re-use the fixture as part of another test, it would be better to just import them across files.